src/eth2.rs

```diff
@@ -290,7 +248,7 @@ pub fn eth2_check(eth2: &str) -> Result<()> {
             }
         }
         _ => {
-            let mut version = "";
```

Comment:
nice i was wondering why i was gettin those warns

Reply:
had to look it up. I guess since we only assign once it doesn't have to be mut